djaoapp/api/users.py

```diff
@@ -40,3 +50,41 @@ def perform_destroy(self, instance):
         if instance.user:
             get_role_model().objects.filter(user=instance.user).delete()
         super(UserProfileAPIView, self).perform_destroy(instance)
+
+
+class RecentActivityAPIView(GenericAPIView):
+    serializer_class = ActivitySerializer
+
+    def get(self, request, *args, **kwargs):
+        start_at_dt = datetime.date.today() - datetime.timedelta(2)
```

Comment:
Use `day_periods` as defined in [djaodjin-saas#199](https://github.com/djaodjin/djaodjin-saas/issues/199)